packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -135,7 +217,7 @@ export class SelectFileName
   }
 }
 
-export class SelectDirPath
+export abstract class SelectDirPath
```

Comment:
Made this an abstract class so the globDirs method has to be implemented. Right now it's implemented by 2 classes. I used inheritance because of composition because the use cases for this seems to be generalized to the following 2 scenarios:
1) List directory paths with paths containing the keyword at the top of the list
2) List directory paths where the keyword is a subdirectory in the path